app/controllers/api/admin/settings_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
+#  Cryptopus and licensed under the Affero General Public License version 3 or later.
+#  See the COPYING file at the top-level directory or at
+#  https://github.com/puzzle/cryptopus.
+
+class Api::Admin::SettingsController < ApiController
```

Comment:
SettingsController? should this be on this branch?
you're working on profile, not settings ...